package.json

```diff
@@ -82,11 +84,14 @@
     "sass-loader": "10.2.0",
     "serve": "12.0.0",
     "standard-version": "9.3.1",
-    "storybook-addon-next-router": "3.0.7",
+    "storybook-addon-next-router": "3.0.8",
     "storybook-css-modules-preset": "1.1.1",
     "style-loader": "2.0.0",
     "typescript": "4.4.2"
   },
+  "resolutions": {
+    "@babel/runtime": "7.15.4"
+  },
```

Comment:
I ended up updating a bunch of packages because there were some incompatibilities causing Storybook to not run.